spec/facility_spec.rb

```diff
@@ -23,4 +31,35 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#register_vehicle' do
+    it 'can register vehicles' do
+
+      # binding.pry
+
+      expect (@facility_1.register_vehicle(@cruz)).should == nil
+      @facility_1.add_service( 'Vehicle Registration' )
+      expect (@facility_1.registered_vehicles).should == []
+      expect (@cruz.plate_type).should == nil
+      expect (@cruz.registration_date).should == nil
+      expect (@facility_1.collected_fees).should == 0
+      expect (@facility_1.register_vehicle(@cruz)).should == [@cruz]
+      # expect (@cruz.plate_type).should == :regular
+      expect (@cruz.registration_date).should == Date.today
+      expect (@facility_1.collected_fees).should == 100
+      # expect(@facility_1.registered_vehicles).should == [@cruz]
+    end
+  end
+
+  describe '#renew_drivers_license' do
+    it 'describes if a license can be renewed' do
+      @registrant_1.license_data[:written] = true
+      @registrant_1.license_data[:license] = true
+      expect(@facility.renew_drivers_license(@registrant_1)).to eq(false)
+      @registrant_1.license_data[:renewed] = false
+      expect(@facility.renew_drivers_license(@registrant_1)).to eq(true)
+      expect(@facility.renew_drivers_license(@registrant_3)).to eq(false)
+      expect(@facility.renew_drivers_license(@registrant_2)).to eq(false)
+    end
+  end
```

Comment:
It looks like you're missing tests for the administer_road_test method -- don't forget to write these tests first to guide your development!

Reply:
administer_written test needs a test too!